### PR TITLE
Fix route.sh and some misc cleanup

### PR DESCRIPTION
**Pull request by anjalidk**

### CODE REVIEW

File: net_route.c [hunk 1]
```diff
@@ -250,6 +250,7 @@ static int handle_route_resp(int net_sock, void **out)
 	struct sockaddr_nl nladdr;
 #ifdef TESTING
 	FILE *fp = NULL;
+	fp = fopen ("./route_info.txt", "w");
```
Comment (hallyn) [inline net_route.c:253]: Should check the return value.

**Comment by anjalidk** [comment]
Done.

**Comment by hallyn** [comment]
Note in general I heavily disapprove of commit messages which say "#fixes".  Each logical change should be documented.

**Comment by anjalidk** [comment]
> On Feb 5, 2024, at 2:15 PM, Serge Hallyn ***@***.***> wrote:
> 
> 
> Note in general I heavily disapprove of commit messages which say "#fixes". Each logical change should be documented.

Ok, will do.

**Pull request merged**